maps/dist/computer.js

```diff
@@ -0,0 +1,18 @@
+import * as Phaser from 'phaser';
```

Comment:
This file should be removed and added to .gitignore

Reply:
i will remove it when the pr will be done.

Reply:
I just saw that you actually want to keep the demo map, so i will gitignore it.